Update usage-estimated-costs.md (#105139)
Azure/azure-content: articles/azure-monitor/usage-estimated-costs.md

### Log Analytics workspace

> To learn about your usage trends and choose the most cost-effective pricing tier (Pay-as-you-go or a [commitment tier](logs/cost-logs.md#commitment-tiers)) for your Log Analytics workspace, select **Usage and estimated costs** from the **Log Analytics workspace** menu in the Azure portal. Note that Usage and estimated costs is **not** showing your actual billed usage. It is calculating what your data ingestion charges would have been for the last 31-days of usage if your workspace had been in each of the available pricing tiers. You can use thses estimated costs to select the lowest cost tier based on your workspace's data ingestion.

Reviewer: ```suggestion
To learn about your usage trends and choose the most cost-effective pricing tier (Pay-as-you-go or a [commitment tier](logs/cost-logs.md#commitment-tiers)) for your Log Analytics workspace, select **Usage and estimated costs** from the **Log Analytics workspace** menu in the Azure portal.

> [!NOTE]
> 
> **Usage and estimated costs** does *not* show your actual billed usage. It calculates what your data ingestion charges would have been for the last 31 days of usage if your workspace had been in each of the available pricing tiers. You can use these estimated costs to select the lowest cost tier based on your workspace's data ingestion.
```
